tests/integration/test_settings_profile/test.py

```diff
@@ -166,21 +166,11 @@ def test_show_profiles():
 
 def test_allow_introspection():
     assert "Not enough privileges" in instance.query_and_get_error("SELECT demangle('a')", user="robin")
-    
-    instance.query("GRANT ALL ON *.* TO robin")
-    assert "Introspection functions are disabled" in instance.query_and_get_error("SELECT demangle('a')", user="robin")
-
-    instance.query("ALTER USER robin SETTINGS allow_introspection_functions=1")
```

Comment:
Why have you shrinked the test that much? Why don't you test previous scenarios?

Reply:
That's good point. I've changed the test to keep some of the previous checks.